lib/ansible/modules/windows/win_mssql_query.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Daniele Lazzari <lazzari@mailup.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub. Actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_mssql_query
+version_added: '2.7'
+short_description: invokes a query on a MSSQL Server
+description:
+    - Invokes a query on a MSSQL Server. This module isn't idempotent, the query will be always executed.
+    - In example, if you run a query that creates a database C(CREATE DATABASE TestDb), the first time it will success
+    - the second time it will fail because the database already exists.
+requirements:
+  - Powershell module SQLPS or Powershell module SqlServer
+options:
+  server_instance:
+    description:
+      - Name of the sql server instance to query.
+    default: computername
```

Comment:
Keep out the default, add a new entry to the description that says `- If not specified, the remote host will be used.`